app/commands/v2/put_link_set.rb

```diff
@@ -13,21 +13,24 @@ def call
         end
 
         if (draft_content_item = DraftContentItem.find_by(content_id: link_params.fetch(:content_id)))
+          draft_payload = Presenters::ContentStorePresenter.present(draft_content_item)
           ContentStoreWorker.perform_async(
             content_store: Adapters::DraftContentStore,
             base_path: draft_content_item.base_path,
-            payload: draft_content_store_payload(draft_content_item),
+            payload: draft_payload,
           )
         end
 
         if (live_content_item = LiveContentItem.find_by(content_id: link_params.fetch(:content_id)))
+          live_payload = Presenters::ContentStorePresenter.present(live_content_item)
           ContentStoreWorker.perform_async(
             content_store: Adapters::ContentStore,
             base_path: live_content_item.base_path,
-            payload: live_content_store_payload(live_content_item),
+            payload: live_payload,
           )
 
-          PublishingAPI.service(:queue_publisher).send_message(message_bus_payload(live_content_item))
+          message_bus_payload = message_bus_payload(live_payload)
```

Comment:
If we're consolidating things through presenters we should probably have a message bus presenter.